resources/AppDelegate.js

```diff
@@ -0,0 +1,6 @@
+let fs = require('fs');
+
+module.exports = function () {
+  fs.copyFile('./resources/AppDelegate.m', './platforms/ios/Vehicle Testing/Classes/AppDelegate.m', () => {
```

Comment:
I'd suggest to stop the build if the file `AppDelegate.m` is not copied correctly:
```suggestion
  fs.copyFile('./resources/AppDelegate.m', './platforms/ios/Vehicle Testing/Classes/AppDelegate.m', (err) => {
    if (err) throw err;
```